README.md

```diff
@@ -189,7 +189,8 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 
 * [cosmos-sdk](https://github.com/cosmos/cosmos-sdk) - A Framework for Building Public Blockchains in the Cosmos Ecosystem.
 * [go-ethereum](https://github.com/ethereum/go-ethereum) - Official Go implementation of the Ethereum protocol.
-* [gossamer](https://github.com/ChainSafe/gossamer) - A Go implementation of the Polkadot Host.
+* [gossamer](https://github.com/ChainSafe/gossamer) - A Go implementation of the Polkadot Host
+* [solana-go](https://github.com/gagliardetto/solana-go) - Go library to interface with Solana JSON RPC and WebSocket interfaces.
```

Comment:
- Golint has many exported functions with no comments
- No coverage report in tests, and I see lots of `no test files` in the repo's test GitHub action.

- I'll see if I can test the coverage locally in a few hours